### PR TITLE
PR for coverage

### PR DESCRIPTION
**Pull request by A-Daneel**
# current coverage: 54,9

Just opening a draft to see how sonarcloud feels about it.

### Controllers:
- Group
- Mail
- Password
- Login

### Command:
- CreateLocalAccount

### CODE REVIEW

File: tests/Functional/Controller/Admin/GroupControllerTest.php [hunk 1]
```diff
@@ -42,55 +48,140 @@ protected function tearDown(): void
...
+        // Arrange
+        $relation = $this->em->getRepository(Relation::class)->findAll()[0];
+        $id = $relation->getId();
+
+        // Act
+        $crawler = $this->client->request('GET', $this->controllerEndpoint . "/relation/delete/{$id}");
+        $form = $crawler->selectButton('Ja, verwijder')->form();
+        $crawler = $this->client->submit($form);
+
+        $allRelations = $this->em->getRepository(Relation::class)->findAll();
```
Comment (jasperweyne) [inline tests/Functional/Controller/Admin/GroupControllerTest.php:181]: I feel like this line can be moved to the `// Arrange` block, eg.
```php
$allRelations = $this->em->getRepository(Relation::class)->findAll();
$relation = $allRelations[0];
$id = $relation->getId();```

**Review by jasperweyne — CHANGES_REQUESTED** [review]
A couple of small suggestions here and there, but other than that, looks good!

File: src/Controller/Security/LoginController.php [hunk 2]
```diff
@@ -58,6 +58,5 @@ public function login_check()
     public function logout()
     {
         // controller can be blank: it will never be executed!
-        throw new \Exception('Don\'t forget to activate logout in security.yaml');
```
[the remark this answers is not in the document]
Comment (A-Daneel) [reply src/Controller/Security/LoginController.php:61]: Think I was still in the part of trying around. sonarcloud got rather mad about it being a too generic exception. However I didn't find any better exception to cover it. 

Any suggestions? then I'll change it.

File: tests/Functional/Controller/Admin/GroupControllerTest.php (hunk 1 above)
Comment (A-Daneel) [reply tests/Functional/Controller/Admin/GroupControllerTest.php:181]: I see what you mean, however I think you misunderstood the flow of the test.

`$allRelations` fetches all the relations after the deletion event to check it does not longer contain the original relation. Thus confirming the deletion was successful.

**Review by jasperweyne — APPROVED** [review]

**Pull request merged**